src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,61 @@
 package edu.kis.vh.nursery;
 
 public class DefaultCountingOutRhymer {
+//TODO-encapsulation methods
 
-    private int[] numbers = new int[12];
+    private static final int CAPACITY = 12;
+    private static final int MAX_INDEX = 11;
+    private static final int EMPTY = -1;
 
-    public int total = -1;
+    private final int[] numbers = new int[CAPACITY];
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = EMPTY;
+    /**
+     * dodaj na stos
+     * @parm in liczba którą chcesz dodać
+     */
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
+    /**
+     *Sprawdz czy stos pusty
+     * @return prawde jesli pusty
+     */
     public boolean callCheck() {
-        return total == -1;
+        return total == EMPTY;
     }
 
+    /**
+     *sprawdz czy stos pełny
+     * @return prawde jeśli pełny
+     *
+     */
     public boolean isFull() {
-        return total == 11;
+        return total == MAX_INDEX;
     }
-
+    /**
+     * zwraca najweksza wartość stosu
+     * @return -1 gdy pusty, liczba jeśli nie
+     */
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return EMPTY;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)